esp/esp/program/modules/handlers/creditcardmodule_stripe.py

```diff
@@ -210,6 +210,8 @@ def charge_payment(self, request, tl, one, two, module, extra, prog):
 
         context = {'postdata': request.POST.copy()}
 
+        group_name = Tag.getTag('full_group_name') or '%s %s' % (settings.INSTITUTION_NAME, settings.ORGANIZATION_SHORT_NAME)
```

Comment:
We may want to just generally have a universal canonical way of getting the full group name, but I think this is fine for now.